src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor().toString().toLowerCase();
```

Comment:
after you fix comment above regarding static method please move initialization of ColorSupplier to class variable as done for Random